shoop/simple_cms/models.py

```diff
@@ -12,13 +12,15 @@
 from django.utils.encoding import force_text, python_2_unicode_compatible
 from django.utils.timezone import now
 from django.utils.translation import ugettext_lazy as _
-from parler.managers import TranslatableQuerySet
+from mptt.managers import TreeManager
+from mptt.models import MPTTModel, TreeForeignKey
+from parler.managers import TranslatableManager
 from parler.models import TranslatableModel, TranslatedFields
 
 from shoop.core.fields import InternalIdentifierField
 
 
-class PageQuerySet(TranslatableQuerySet):
+class PageManager(TranslatableManager, TreeManager):
```

Comment:
OK, it seems that there is no such thing. But you can still let the PageQuerySet be as a queryset by using `from_queryset` (see https://docs.djangoproject.com/en/1.9/topics/db/managers/#from-queryset).

```
class PageQuerySet(TranslatableQuerySet):  # as before
    ...


class Page(MPTTModel, TranslatableModel):
    ...
    objects = TreeManager.from_queryset(PageQuerySet)()  # this is the trick!
```
```